src/interceptors/ClientRequest/utils/normalizeHttpRequestParams.ts

```diff
@@ -26,7 +26,7 @@ function resolveRequestOptions(
   /**
    * Clone the request options to lock their state
    * at the moment they are provided to `ClientRequest.
-   * @see https://github.com/mswjs/node-request-interceptor/issues/86
+   * @see https://github.com/@mswjs/interceptors/issues/86
```

Comment:
You don't have to include `@` in the organization name. This should be `mswjs/interceptors` instead.

Reply:
Bad find and replace on my part 🤦‍♂️. Will update.